pkgs/by-name/mn/mnemosyne/package.nix

```diff
@@ -1,49 +1,48 @@
 {
   python3Packages,
   fetchurl,
-  anki,
+  qt6,
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "mnemosyne";
   version = "2.10.1";
-  format = "setuptools";
+  pyproject = true;
 
   src = fetchurl {
     url = "mirror://sourceforge/project/mnemosyne-proj/mnemosyne/mnemosyne-${version}/Mnemosyne-${version}.tar.gz";
     sha256 = "sha256-zI79iuRXb5S0Y87KfdG+HKc0XVNQOAcBR7Zt/OdaBP4=";
   };
 
-  nativeBuildInputs = with python3Packages; [ pyqtwebengine.wrapQtAppsHook ];
```

Comment:
This was just an alias for `libsForQt5.wrapQtAppsHook` which is incorrect, since this package uses qt6.